app/src/main/java/com/android/voyageur/ui/trip/activities/Activities.kt

```diff
@@ -32,9 +74,204 @@ fun ActivitiesScreen(
             selectedItem = navigationActions.currentRoute())
       },
       content = { pd ->
-        Text(
-            modifier = Modifier.padding(pd).testTag("emptyActivitiesPrompt"),
-            text =
-                "You're viewing the Activities screen for ${trip.name}, but it's not implemented yet.")
+        LazyColumn(
+            modifier =
+                Modifier.padding(pd).padding(top = 16.dp).fillMaxWidth().testTag("lazyColumn"),
+            verticalArrangement = Arrangement.spacedBy(10.dp, Alignment.Top),
+        ) {
+          item {
+            Text(
+                text = "Drafts",
+                fontSize = 24.sp,
+                fontWeight = FontWeight.Bold,
+                modifier = Modifier.padding(start = 10.dp))
+          }
+          drafts.forEach { activity ->
+            item {
+              ActivityItem(activity)
+              Spacer(modifier = Modifier.height(10.dp))
+            }
+          }
```

Comment:
Not, really an issue, but I believe you could also use the items function.

Reply:
Thanks for the suggestion! For now I think that not using the function makes the code a bit more readable, and there's not too much duplicated code since there are only two sections (Drafts and Final). If you agree, I would leave it like this for now